### ISSUE TITLE
Difference in colour between AVPro and Unity video player

### ISSUE THREAD

**Issue by Mildebug** [RenderHeads/UnityPlugin-AVProVideo#929]
**Describe the issue**
Hello, I've noticed a difference in colour between the AVPro player and the Unity video player.
I've already tried to change the video api from MediaFoundation to other but it doesn't seems to work.
I've also tried to deactivate the hardware encoding.

I'm in gamma color space in the project settings of Unity.

Do you have any idea ? 

Thanks !

**Your Setup (please complete the following information):**
 - Unity version: 2021.1.12f1
 - AVPro Video version: v2.1.6
 - Operating system version: Windows 10
 - Device model: Unity Editor
 - Video specs (resolution, frame-rate, codec, file size): 1920x1080, 25, VP8 codec, 130Mo

**To Reproduce**
1. 
2. 
3. 

**Logs**
If applicable, add error logs to help explain your problem.

**Screenshots**
![image](https://user-images.githubusercontent.com/90604952/133057136-539f89b7-2cfe-409c-b082-5fa5be3e881f.png)

**Videos**
If applicable, add a copy of your video or the URL

### Please DO NOT LINK / ATTACH YOUR PROJECT FILES HERE
Instead email the link to us unitysupport@renderheads.com

**Comment by kahnivore** [comment]
Regarding the video only playing with  Media Foundation API see [these notes](https://www.renderheads.com/content/docs/AVProVideo/articles/supportedmedia.html#video-codecs) on VP8 support. 

That's interesting. What is your setup? Are you using one of the demo scenes? Displaying as UI or on mesh? Which shader are you using? 

Which one between AVPro Video and Unity Video Player is showing the desired output/original colour? 

Would it be possible to send your video to unitysupport@renderheads.com so we could test it? If you do, please add #929 to the subject line.

**Comment by Mildebug** [comment]
No, I'm not using a demo scene and I'm not displaying on the Display uGUI component or a mesh.
I extract the texture from the TextureProducer of the MediaPlayer and I apply it to the backgroundImage property of a VisualElement of UIToolKit.
I tried to compare with the render result of the Display uGUI (with default parameters and shader) but I've got the same result as in the VisualElement.

I'm not using a particular shader.

**Comment by kahnivore** [comment]
Could you please send through the media you're testing with?

**Comment by kahnivore** [comment]
Thanks for sending the video, we have managed to reproduce the colour discrepancy. This might be something we can't do anything about - i.e. it's just how Microsoft's decoder is handling it.

We'd need to investigate a bit more to determine the exact cause because it could be a variety of things but suspect it won't happen with VP9 if that could be a workable solution for you in the meantime.
